### DISCUSSION TITLE
element-团队解决issue的能力缓慢啊，社区人员这么多给你免费测试参与提意见，issue过了好久都没人响应，请问对的起大家吗

### DISCUSSION THREAD

**Discussion by tianhailiang** [element-plus/element-plus#15800] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `All`

## Description

element-团队解决issue的能力缓慢啊，社区人员这么多给你免费测试参与提意见，issue过了好久都没人响应，请问对的起大家吗

### Additional comments

element-团队解决issue的能力缓慢啊，社区人员这么多给你免费测试参与提意见，issue过了好久都没人响应，请问对的起大家吗

**Comment by AlwaysNoobCoder** [comment]
白嫖还嫌慢？ 别人也是有自己的工作和生活的，不是人人都生活在社会福利溢出的北欧的。

很多用了开源项目的人，有想过回来回馈社区吗（比如帮助解答小白的问题）？

**Reply by JucsyLucy** [reply]
一针见血的
